### ISSUE TITLE
Option to display only studies from a specific lab

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#620]
# TL;DR
Provide a way to see only studies from a given lab on Experiment.

# Narrative
As a staff member, it's helpful to be able to see what other studies a lab is running when reviewing one study or corresponding with lab members. 

As a researcher, it's helpful to see studies grouped by lab in the case of the Sandbox vs. Demo vs. actual labs.

# Acceptance Criteria
A precise description of how/when this issue will be considered closed, e.g.:

- [x] Add a study list view with custom URL slug showing only that lab's (active, discoverable) studies
- [x] New column on lab object for custom url; initially populated with the lab's display name reduced to [a-z0-9] plus hyphen
- [x] New field on the lab detail page https://lookit.mit.edu/exp/labs/n/ below Lab Website, named "Custom URL", and appearing as "https://lookit.mit.edu/studies/mycustomslug" (or whatever the right address for the search is)
- [x] New field added to Edit Lab page, below Lab Website.  Add descriptions below each field so it looks like this:

##### Lab Website
[Text entry field w/ current value]
*A link to an external website, such as your university lab page*

##### Custom URL
https://lookit.mit.edu/studies/ [Text entry field]
*A unique URL (https://lookit.mit.edu/studies/mylabname) that will show discoverable, active studies for this lab only*

- [x] The Custom URL field on the edit page should enforce uniqueness however the Lab Name field enforces it

- [x] Test that correct studies show up in filtered study list - should show only discoverable, active studies as with the normal study view page.

**Comment by mekline** [comment]
@okaycj added details to the issue above! For the custom slug, I think we need to spell out how this will appear in the full URL, either by including the stem before the text box as I did, or by including the example in the help text below.  

Another option would be to put the whole URL in this field and show an error if someone edits it to not be a legal URL for a custom search, but I worry that will just be confusing.

Happy to follow your intuition on which of these is clearest!
